### PR TITLE
Example turtle and figures are updated

### PR DESCRIPTION
**Pull request by rajaram5**
I have updated the example turtle files further. I replaced some of wrong sio URIs and I have updated turtle for some modules. The figures also updated to reflect example rdf changes

### CODE REVIEW

**Comment by NuriaQueralt** [comment]
> I have updated the example turtle files further. I replaced some of wrong sio URIs and I have updated turtle for some modules. The figures also updated to reflect example rdf changes

Hi @rajaram5 I also updated the person.shex. I noticed that you moved 'my' person.ttl to sio-model/ folder. Could you do the same with the person.shex for consistency bf merging the PR pls?

**Comment by rajaram5** [comment]
@NuriaQueralt I fixed the PR issues. Can you have a look at it please?

**Review by sxzhang1201 — APPROVED** [review]
Thanks for 1) distinguishing CDE examples and CDE-SIO example 2) updating figures!

File: shex/sio-model/personShape.shex [hunk 1]
```diff
@@ -0,0 +1,18 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
```
Comment (sxzhang1201) [inline shex/sio-model/personShape.shex:1]: Hi @rajaram5, is the version `/v010/` correct?

Comment (rajaram5) [reply shex/sio-model/personShape.shex:1]: Nice catch, I will fix version in the IRI

**Pull request merged**